docs/howto/sqlite-json1.txt

```diff
@@ -0,0 +1,87 @@
+=======================================
+How to enable JSON1 extension on SQLite
+=======================================
+
+This document describes how to enable JSON1 extension on Python's
+:py:mod:`sqlite3` library so that :class:`~django.db.models.JSONField`
+can be used. To check if the extension is enabled on your installation, you can
+do a query with one of the functions included in the extension, e.g.
+``JSON()``. For example::
+
+    >>> import sqlite3
+    >>> conn = sqlite3.connect(':memory:')
+    >>> c = conn.cursor()
+    >>> c.execute("SELECT JSON('123');")
+
+If the query doesn't throw any errors, then the JSON1 extension should already
+be enabled. Otherwise, follow the instructions below according to your
+operating system to set it up correctly.
```

Comment:
Before mentioning how to test in a Python console, you could mention first that the field check will fail. That would also help searchability since if people often search just for the error message they see :)

Reply:
I've changed it. Not sure if I should include the error message, though..